src/main/kotlin/game/LottoGame.kt

```diff
@@ -0,0 +1,44 @@
+package game
+
+import domain.*
+import model.PurchaseLottoInfo
+import ui.InputManager
+import ui.OutputManager
+
+class LottoGame(
+    private val inputManager: InputManager = InputManager(),
+    private val outputManager: OutputManager = OutputManager(),
+    private val lottoGenerator: LottoGenerator = LottoGenerator()
+) {
+    fun startLottoGame() {
+        with(outputManager) {
```

Comment:
```suggestion
    fun startLottoGame() = with(outputManager) {
```
이렇게 처리하면 가독성이 더 좋을 것 같습니다.